### DISCUSSION TITLE
name change

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [open-audio-stack/open-audio-stack-core#10] [General]
@kmturley 

are you sure you want to stick with "open audio" as the name for this? it's very generic

**Comment by kmturley** [comment]
Name was a placeholder, because I couldn't think of a good one. Open to suggestions!

**Comment by RustoMCSpit** [comment]
a portmanteau of studiorack and owlplug? your name? whatever it is, a d*main has to be got instantly. it could be something meaningless that sounds techy like reonim or ReoStudio

**Comment by kmturley** [comment]
StudioRack was named like a real company's product, with polished marketing site and branding. This led some open-source folk to not trust it. They didn't believe a single person made it all and that it was free. They thought it would go paid at some point and were reluctant to support/use it. For an open-source project to gain adoption it doesn't need to be perfect.

**Reply by RustoMCSpit** [reply]
not perfect but SEO optimised, it has to be more distinct. SourceWell? DanceWizard? MagicOven?
